data/database.py

```diff
@@ -755,6 +756,26 @@ class RobotAccountToken(BaseModel):
     fully_migrated = BooleanField(default=False)
 
 
+class QuotaLimitGroups(BaseModel):
+    group_name = CharField()
+
+
+class QuotaType(BaseModel):
+    description = CharField()
```

Comment:
Shouldn't this column be named "name" instead.
It will also need to be prepopulated with possible values as part of the migration.

Reply:
This makes sense to rename and it is already being prepopulated by the alembic upgrade.